### DISCUSSION TITLE
Can I disable PH Lockout

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#968] [REM]
PH reading is working right now, so I have to test PH manually.

I'd like chlorine to keep dosing, but PH lockout is preventing chlorine from dosing.

Is there a way to disable PH lockout?

**Comment by rstrouse** [comment]
Uncheck use chlorinator then change the pH lockout to a larger number then check the use chlorinator again.  It appears that the pH lockout is not visible when the use chlorinator is checked.

**Reply by davthomaspilot** [reply]
I've changed the PH lock to the maximum 8.4.  But, my malfunctioning ph probe (or something) is making ph read over 12.

I need to totally disable it.
